test/image/mocks/pie_title_pull.json

```diff
@@ -0,0 +1,21 @@
+{
+  "data": [
+    {
+      "values": [
+        50, 49, 48, 47, 46, 45, 44, 43, 42, 41
+      ],
+      "pull": [
+        0, 0.5, 0, 0.5, 0, 0.5, 0, 0.5, 0, 0.5
```

Comment:
Thanks for considering `pull` in all this - nice touch!

Reply:
Thanks!